modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/apirequest/ApiRequestEnrichment.scala

```diff
@@ -40,7 +40,6 @@ object ApiRequestEnrichment extends ParseableEnrichment {
       "api_request_enrichment_config",
       "jsonschema",
       1,
-      0,
```

Comment:
I wondered initially if the criteria should be `valid == 1-0-0 or 1-0-1` rather than `valid == 1-0-x`

but it seems like ApiRequestEnrichment was the outlier here. See: [`WeatherEnrichement`](https://github.com/snowplow/enrich/blob/65d7f05e6937393bf983ed91467b5c7dcac3d34d/modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/WeatherEnrichment.scala#L49), [`YauaaEnrichement`](https://github.com/snowplow/enrich/blob/481760213049a86c84099c33679293c0af9aef0e/modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/YauaaEnrichment.scala#L38), [`HttpHeaderExtractorEnrichment`](https://github.com/snowplow/enrich/blob/65d7f05e6937393bf983ed91467b5c7dcac3d34d/modules/common/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/registry/HttpHeaderExtractorEnrichment.scala#L33)